lib/pbench/agent/tool_data_sink.py

```diff
@@ -21,12 +21,18 @@
 import subprocess
 import sys
 import tempfile
+
+from configparser import DuplicateSectionError
+from daemon import DaemonContext
+from jinja2 import Environment, FileSystemLoader
+from pathlib import Path
 from threading import Thread, Lock, Condition
 from typing import Any, Dict, List, NamedTuple, Tuple
 
 from bottle import Bottle, ServerAdapter, request, abort
 from configparser import DuplicateSectionError
 from daemon import DaemonContext
+from distutils.spawn import find_executable
```

Comment:
LGTM points out that the code is not using this `import`...which is a good thing, since we're switching away from this function....

Reply:
The merge process must have missed this change.